libphal/phal_sbe.C

```diff
@@ -229,30 +244,14 @@ sbeError_t captureFFDC(struct pdbg_target *target)
 	uint32_t ffdcLen = 0;
 	uint32_t status = 0;
 
-	if (is_ody_ocmb_chip(target)) {
-		// sbe_ffdc_get internall uses chipop target so probe it
-		pdbg_target *co = get_ody_chipop_target(target);
-		if (pdbg_target_probe(co) != PDBG_TARGET_ENABLED) {
-			return sbeError_t(exception::PDBG_TARGET_INVALID);
-		}
-		// TODO at present OCMB is under proc and it does not have its
-		// own pib target in device tree. This needs to be fixed after
-		// ocmb is pulled out of proc to use ocmb pib target directly
-		if (sbe_ffdc_get(target, &status, bufPtr.getPtr(), &ffdcLen)) {
-			log(level::ERROR,
-			    "odyssey sbe_ffdc_get function failed");
-			throw sbeError_t(exception::SBE_FFDC_GET_FAILED);
-		}
-	} else {
-		// Log SBE debug data.
-		logSbeDebugData(target);
+	// Log SBE debug data.
+	logSbeDebugData(target);
 
-		// get PIB target
-		pdbg_target *pib = getPibTarget(target);
-		if (sbe_ffdc_get(pib, &status, bufPtr.getPtr(), &ffdcLen)) {
-			log(level::ERROR, "proc sbe_ffdc_get function failed");
-			throw sbeError_t(exception::SBE_FFDC_GET_FAILED);
-		}
+	// get PIB target
+	pdbg_target *pib = getPibTarget(target);
```

Comment:
Better to have a check for `nullptr` of the returned `pib` target before we call `sbe_ffdc_get`